environment:
    matrix:
    - TARGET_ARCH: x86
      CONDA_PY: 3.4
      CONDA_INSTALL_LOCN: C:\\Miniconda3

    - TARGET_ARCH: x64
      CONDA_PY: 3.4
      CONDA_INSTALL_LOCN: C:\\Miniconda3-x64

    - TARGET_ARCH: x86
      CONDA_PY: 3.5
      CONDA_INSTALL_LOCN: C:\\Miniconda35
    
    # x64 py3.5 and py3.6 disabled until condas scipy build is fixed, see https://github.com/ContinuumIO/anaconda-issues/issues/1415
    # - TARGET_ARCH: x64
    #   CONDA_PY: 3.5
    #   CONDA_INSTALL_LOCN: C:\\Miniconda35-x64

    - TARGET_ARCH: x86
      CONDA_PY: 3.6
      CONDA_INSTALL_LOCN: C:\\Miniconda

    # - TARGET_ARCH: x64
    #   CONDA_PY: 3.6
    #   CONDA_INSTALL_LOCN: C:\\Miniconda-x64

init:
  - ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%

install:
  - "set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\\Scripts;%PATH%"
  - conda config --set always_yes yes --set changeps1 no
  - conda update -q conda
  - conda config --append channels conda-forge
  - conda info -a
  - "conda create -q -n test-environment python=%CONDA_PY% numpy scipy plotly pytest"
  - activate test-environment

build: off

test_script:
  - pytest -v
